biz.aQute.resolve/src/biz/aQute/resolve/BndrunResolveContext.java

```diff
@@ -187,7 +187,7 @@ public synchronized void init() {
 
 				Parameters systemPackages = new Parameters(properties.mergeProperties(Constants.RUNSYSTEMPACKAGES),
 						project);
-				system.addExportPackages(systemPackages);
+				system.addSystemBundleExportPackages(systemPackages);
```

Comment:
This is the new deprecated method. Use a non-deprecated method here.

Reply:
It's not clear what non-deprecated method I can invoke. I don't know the bundle identity/version. I can just default to `system.bundle`/empty here but then that defaulting is duplicated here and in `ResourceBuilder`.